src/sidebar/styles.css

```diff
@@ -177,3 +179,7 @@ footer {
 .forbid-cursor {
   cursor: no-drop;
 }
+
+.mdl-js-button {
+  height: 100%;
+}
```

Comment:
Again, sorry,  not super familiar w/ Material Design, but this feels pretty global and specific and could cause some future 🔥.

Should we add a more generic class name here as well so we can style _just_ that one element instead of all buttons globally (even though we only use one at the moment)?

```jsx
<button id="demo-menu-top-right" class="mdl-js-button more-button">
  <i class="material-icons">more_vert</i>
</button>
```